src/vis/vis-view.js

```diff
@@ -33,6 +33,8 @@ var Vis = View.extend({
 
     this.model.once('load', this.render, this);
     this.model.on('invalidateSize', this._invalidateSize, this);
+    this.model.overlaysCollection.on('add remove', this._addOverlays, this);
```

Comment:
I see. Then I think we could rename it `this._resetOverlays` or something a bit more meaningful.